### DISCUSSION TITLE
考虑做成 Python GUI 不 ?

### DISCUSSION THREAD

**Discussion by monkeycc** [DouyinFE/semi-design#1894] [General]
https://github.com/flet-dev/flet
Flutter 都能做成python形式

我觉得semi-design应该也行

**Comment by pointhalo** [comment]
我们自身业务没有这种诉求，因此没有人力去做这方面的投入😂

**Comment by SudoUserReal** [comment]
做机器学习的都用 gradio，比如 stable diffusion webui  和 text generate webui，不做机器学习的话，用 python 画 GUI 属实难受场景太少，不如 QT，长短期来看需求都不太够
